python/lsst/ts/ATWhiteLightSource/fakeChillerComponent.py

```diff
@@ -41,8 +43,7 @@ async def send_command(self, cmd):
             await asyncio.sleep(0.05)
```

Comment:
Is there a reason for this sleep here? might worth adding a comment here to explain why you added it.

Reply:
I agree. It's also disturbing to have a 1/2 second hard coded sleep; a named module-level constant would be better with a comment explaining what it's for.

Reply:
it was just to simulate a bit of tcp latency. I removed it.